moto/medialive/urls.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import unicode_literals
+from .responses import MediaLiveResponse
+
+url_bases = [
+    "https?://medialive.(.+).amazonaws.com",
+]
+
+
+response = MediaLiveResponse()
+
+
+url_paths = {
+    "{0}/.*$": response.dispatch,
```

Comment:
You'll need to add paths for `/prod/channels`, `/prod/inputs`, etc. for this to work in moto server mode.  See `urls.py` for other services for examples (`polly` is similar).